### DISCUSSION TITLE
Clicking number doesn't work

### DISCUSSION THREAD

**Discussion by Nomes77** [JohnyP36/Connect4#35] [Bug report] [answered]
### Prerequisites

- [X] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
- [X] This is not a question. -- otherwise use this [template](https://github.com/JohnyP36/Connect4/issues/new?assignees=&labels=question&template=questions.yml)

### Description of the bug

When clicking number it should open a issue
https://github.com/JohnyP36/Connect4/issues/new?title=connect4%7Cdrop%7Cred%7C4&body=Just+push+%27Submit+new+issue%27+without+editing+the+title.+The+README+will+be+updated+after+approximately+30+seconds.
However it does not

### Current behaviour

Ask to choose template

### Expected behaviour

Should go te link/issue clicked

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

**Comment by JohnyP36** [comment, marked as answer]
Does this work? https://github.com/JohnyP36/Connect4/commit/a07ae9f766404aff3d0f6a73863b12ef52ac9f15

**Comment by Nomes77** [comment]
Yes it does work!
